pkg/ts/utils/object.ts

```diff
@@ -67,7 +67,7 @@ export function deepCopy<T>(datum: T): T {
   if (!datum) {
     return datum;
   } else if (Array.isArray(datum)) {
-    // This is trivially type safe but tsc cannot prove it so we have to 'promise'.
+    // This is trivially type safe but tsc needs help
```

Comment:
Not sure what the problem was here, unless it was too cheeky :)

Reply:
No, I just stumbled over it because I started looking for Promise.